update/protocol.go

```diff
@@ -35,6 +35,8 @@ type Update struct {
 	Type         Type    `codec:"type" json:"type"`
 	PublishedAt  *Time   `codec:"publishedAt,omitempty" json:"publishedAt,omitempty"`
 	Asset        *Asset  `codec:"asset,omitempty" json:"asset,omitempty"`
+	DokanCodeX86 string  `codec:"dokanCodeX86" json:"dokanCodeX86,omitempty"`
+	DokanCodeX64 string  `codec:"dokanCodeX64" json:"dokanCodeX64,omitempty"`
```

Comment:
I think windows-specific driver information probably belongs one level down in the update structure.  What about:

```go

type Update struct {
    ...
     WindowsDrivers *WindowsDrivers `codec:"windowsDrivers,omitempty" json:windowsDrivers,omitempty"`
    ...
}

type WindowsDrivers struct {
    DokanCodeX86  string
    DokanCodeX64  string
}
```